postcss.config.js

```diff
@@ -0,0 +1,37 @@
+export default {
+    plugins: {
+        tailwindcss: {},
+        autoprefixer: {},
+        'postcss-pxtorem': {
+            rootValue: 16,
+            unitPrecision: 5,
+            propList: [
+                'font',
+                'font-size',
+                'line-height',
+                'letter-spacing',
+                'border*',
+                'background*',
+                'grid*',
+                'top',
+                'left',
+                'bottom',
+                'right',
+                'inset',
+                'width',
+                'height',
+                'margin*',
+                'padding*',
+                'max-*',
+                'min-*',
+                'gap*',
+            ],
```

Comment:
Is this entirely necessary? We'll need to keep it upgraded based on props. We'll want to include `size-*` for instance.

Reply:
This is just my running list, but if there are other CSS properties that might have pixel values we should add them.

Reply:
Personally I hate using functions like `remPair` and `pxPair` and the like. I prefer using the values we get from designers in Figma and letting PostCSS handle the conversion. I mean, this is what I'd add to projects I'm on, but ultimately it’s up to UI, I guess.

Reply:
@ekfuhrmann 
Ok, so we could use `propList: [*]` and not need to update this; as I was adding postcss-pxtorem to Viget projects, I wanted to be cautious and know what was being changed. 

I think we have two options: 
- an explicit list of properties
- a wildcard propList

Reply:
Generally I use the following settings which seems to work best for me:

```
{
  rootValue: 16,
  propList: ['*'],
  mediaQuery: true,
  minPixelValue: 0,
},
```

Reply:
Is `propList` referring to CSS properties? 

`size-` is just a Tailwind class right? 

I'm cool with using a `*` and letting projects opt out of specific properties. 

[From the docs](https://github.com/cuth/postcss-pxtorem?tab=readme-ov-file#options)

> Use ! to not match a property. Example: ['*', '!letter-spacing']